plugins/board-resources/src/components/KanbanCard.svelte

```diff
@@ -37,7 +37,7 @@
   }
 </script>
 
-<div class="card-container" {draggable} class:draggable on:dragstart on:dragend>
+<div class="card-container" {draggable} class:draggable on:dragstart on:dragend class:dragged={dragged}>
```

Comment:
I think we should extract base Kanban Card component

Reply:
Will do next step :-)